apifest/src/main/java/com/apifest/HttpRequestHandler.java

```diff
@@ -117,7 +121,11 @@ public void messageReceived(ChannelHandlerContext ctx, MessageEvent e) {
                             break;
                         }
                     }
-
+                    //If there is no access token in request header, try to find it in request body
+                    //This is used for authorized file download requests when authorization header is not provided 
+                    if (accessToken == null) {
```

Comment:
also check if content type is urlencoded as defined in [RFC](http://tools.ietf.org/html/rfc6750#section-2.2).